test/einops_comparison.jl

```diff
@@ -0,0 +1,182 @@
+""" These tests are inspired by the paper published on python's "einops" package.
+
+See 'Alex Rogozhnikov, 2022. "EINOPS: CLEAR AND RELIABLE TENSOR MANIPULATIONS WITH EINSTEIN-LIKE NOTATION",
+published at ICLR 2022'; Listing 1.
+https://openreview.net/pdf?id=oapKSVM2bcj
+"""
+
```

Comment:
Maybe it's nice to make this file self-contained, for anyone reading it as documentation. Also, loading Compat to use `stack` below:
```suggestion

using TensorCast, Test, Statistics, Compat

```

Reply:
Nice, but defined more clearly in 2b37d6501c886e69727006e22ba98815e37a5d27.

Reply:
That's great, even clearer.